CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.1.0]
+### Changed
+- Confirm a transaction conditionally ([#301](https://github.com/MetaMask/smart-transactions-controller/pull/301))
+- meta: Drop nodejs v16 support ([#277](https://github.com/MetaMask/smart-transactions-controller/pull/277))
+
+### Fixed
+- Update the "doesTransactionNeedConfirmation" function, add additional tests ([#307](https://github.com/MetaMask/smart-transactions-controller/pull/307))
```

Comment:
This method is private, so consumers aren't able to see it. 

Also, how was this method updated? Saying that it was updated doesn't seem helpful to understand the real change here.

Based on the tests that were added, perhaps we want something like:

``` suggestion
- Ensure that a non-regular transaction can be confirmed ([#307](https://github.com/MetaMask/smart-transactions-controller/pull/307))
- Ensure that a transaction that does not have a `minedHash` can be confirmed ([#307](https://github.com/MetaMask/smart-transactions-controller/pull/307))
```

But also, this PR seems to build upon 301. In fact the only real change here was to add new tests — the behavior described above should have already worked, it doesn't seem like the PR fixed a pre-existing problem. So perhaps we don't need either of these entries?

``` suggestion
```